Gems/AtomTressFX/Code/Passes/HairSkinningComputePass.cpp

```diff
@@ -225,8 +232,13 @@ namespace AZ
                 // This includes the PerView, PerScene and PerPass srgs (what about per draw?)
                 SetSrgsForDispatch(commandList);
 
-                for (const RHI::DispatchItem* dispatchItem : m_dispatchItems)
+                AZStd::unordered_set<const RHI::DispatchItem*>::iterator it = m_dispatchItems.begin();
+                AZStd::advance(it, context.GetSubmitRange().m_startIndex);
+
+                for (uint32_t index = context.GetSubmitRange().m_startIndex; index < context.GetSubmitRange().m_endIndex; ++index, ++it)
```

Comment:
A quick comment to say that we are submitting items across the range provided by RHI